server/src/main/java/org/opensearch/indices/recovery/RecoveryTarget.java

```diff
@@ -177,51 +177,6 @@ public boolean reset(CancellableThreads newTargetCancellableThreads) throws IOEx
         return false;
     }
 
-    /**
-     * cancel the recovery. calling this method will clean temporary files and release the store
-     * unless this object is in use (in which case it will be cleaned once all ongoing users call
-     * {@link #decRef()}
-     * <p>
-     * if {@link #cancellableThreads()} was used, the threads will be interrupted.
-     */
-    public void cancel(String reason) {
-        if (finished.compareAndSet(false, true)) {
-            try {
-                logger.debug("recovery canceled (reason: [{}])", reason);
```

Comment:
Nitpick - since we're now invoking the superclass method:

https://github.com/opensearch-project/OpenSearch/blob/197909c71f91f1e26da7bc6c7218b53fbb73cdae/server/src/main/java/org/opensearch/indices/replication/common/ReplicationTarget.java#L156-L163

perhaps the log statement should be more generic? `Replication/Recovery cancelled` or something similar?